app/containers/HomeContainer/selectors.js

```diff
@@ -1,29 +1,29 @@
 import { createSelector } from 'reselect';
 import get from 'lodash/get';
-import { initialState } from './reducer';
+import { iTunesServiceInitialState } from './reducer';
 
 /**
  * Direct selector to the homeContainer state domain
  */
-
-const selectHomeContainerDomain = (state) => state.homeContainer || initialState;
-
 /**
  * Other specific selectors
  */
 
 /**
  * Default selector used by HomeContainer
  */
+// is this wrong? https://redux.js.org/usage/deriving-data-selectors#createselector-overview. Read the Warning
+// we have DRY breakage here...
+//export const selectHomeContainer = () => createSelector(selectHomeContainerDomain, (substate) => substate);
+export const selectHomeContainer = (state) => state.homeContainer || iTunesServiceInitialState;
```

Comment:
makes sense, please make the change in react-floki as well, so it generates it correctly

Reply:
👍🏼